src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs

```diff
@@ -56,6 +54,26 @@ internal DnnImageFeaturizerInput(string outputColumnName, string inputColumnName
     /// | Input column data type | Vector of <xref:System.Single> |
     /// | Output column data type | Vector of <xref:System.Single>, the size of the vector depends on the pre-trained DNN |
     ///
+    /// NuGet requirements:
+    ///	- Microsoft.ML.OnnxTransformer
+    /// - Microsoft.ML.OnnxRuntime(question: does OnnxTransformer already pull OnnxRuntime as dependency? if so we don't need to add this)
```

Comment:
>- Microsoft.ML.OnnxRuntime(question: does OnnxTransformer already pull OnnxRuntime as dependency? if so we don't need to add this) [](start = 8, length = 130)

please remove Microsoft.ML.OnnxRuntime requirement. It's already included in Microsoft.ML.OnnxTransformer